user_guide_src/source/incoming/routing.rst

```diff
@@ -85,7 +85,8 @@ Specifying Route Handlers
 Controller's Namespace
 ----------------------
 
-If a controller name is stated without beginning with ``\``, the :ref:`routing-default-namespace` will be prepended:
+When you specify a controller and method name as a string,
+if a controller name is stated without beginning with ``\``, the :ref:`routing-default-namespace` will be prepended:
```

Comment:
I do not understand what is "stated without beginning". Well, I understand this means not fully qualified but the wordings are not clear enough.

Reply:
I want to say 
- `App\Controllers\Foo` (without beginning with ``\``) is treated as *default namespace* (`App\Controllers`) + `App\Controllers\Foo` (= `\App\Controllers\App\Controllers\Foo`), 
- but `\App\Controllers\Foo` is treated as ``\App\Controllers\Foo``

Reply:
Maybe "if a controller is written without a leading \``\\``"?